diff --git a/apps/childcount/forms/general.py b/apps/childcount/forms/general.py
index c9b4292..ca3e491 100755
--- a/apps/childcount/forms/general.py
+++ b/apps/childcount/forms/general.py
@@ -63,4 +63,10 @@ class MessageForm(BaseForm):
 class CaseForm(BaseModelForm):
     class Meta:
         model = Case
-        exclude = ('created_at', 'updated_at')
\ No newline at end of file
+        exclude = ('created_at', 'updated_at')
+
+class CaseFormNew(BaseModelForm):
+    class Meta:
+        model = Case
+        exclude = ('created_at', 'updated_at','ref_id','estimated_dob','guardian_id')
+
diff --git a/apps/childcount/reports.py b/apps/childcount/reports.py
index 7f9ef1a..0e3f85a 100755
--- a/apps/childcount/reports.py
+++ b/apps/childcount/reports.py
@@ -5,7 +5,7 @@ from rapidsms.webui.utils import render_to_response
 from django.db.models import ObjectDoesNotExist, Q
 from django.contrib.auth.models import User, Group
 from datetime import datetime, timedelta
-from childcount.forms.general import MessageForm
+from childcount.forms.general import MessageForm,CaseFormNew
 #from pygsm.gsmmodem import GsmModem
 
 
@@ -688,4 +688,53 @@ def trend(request, object_id=None, per_page="0", rformat="pdf"):
             
             pdfrpt.setTableData(queryset, fields, c.get_name_display())
     
-    return pdfrpt.render()
\ No newline at end of file
+    return pdfrpt.render()
+@login_required
+def childcount(request):
+    template_name="childcount/reports/childcount.html"
+    
+    
+    return render_to_response(request, template_name)
+
+def saisie(request):
+    template_name="childcount/formulaire/saisie.html"
+
+
+    if request.method == 'POST': # If the form has been submitted...
+        form = CaseFormNew(request.POST) # A form bound to the POST data
+        if form.is_valid(): # All validation rules pass
+            # Process the data in form.cleaned_data
+            # ...
+            prenom = form.cleaned_data['first_name']
+            """
+            prenom = form.cleaned_data['first_name']
+            nom = form.cleaned_data['last_name']
+            sexe = form.cleaned_data['gender']
+            dob = form.cleaned_data['dob']
+            mere = form.cleaned_data['guardian']
+            mobile = form.cleaned_data['mobile']
+            reporter = form.cleaned_data['reporter']
+            location = form.cleaned_data['location']
+            statut = form.cleaned_data['status']
+
+            enfant = Case(first_name = prenom, last_name = nom,    gender=sexe,dob=dob,guardian=mere,mobile=mobile,reporter=reporter,location=location,status=statut)
+            #Et l'enregister
+            enfant.save() 
+            """
+            enfant = form.save(commit=False)
+            enfant.title = prenom
+            enfant.save()            
+            #form.save
+        last_ten   = Case.objects.order_by('-created_at')[:10] 
+        form = CaseFormNew()      
+    #return HttpResponseRedirect('childcount/saisie') # Redirect after POST
+    else:
+        last_ten   = Case.objects.order_by('-created_at')[:10]
+        form = CaseFormNew() # An unbound form
+
+    return render_to_response(request,template_name, {
+       'form': form,
+       'last_ten': last_ten,
+    })
+    
+
diff --git a/apps/childcount/templates/childcount/reports/reports.html b/apps/childcount/templates/childcount/reports/reports.html
index bf63b66..2c8989e 100644
--- a/apps/childcount/templates/childcount/reports/reports.html
+++ b/apps/childcount/templates/childcount/reports/reports.html
@@ -1,7 +1,12 @@
 {% extends base_template %}
 {% block title %}{{ app.name }}{% endblock %}
-
-
+{% block page_tabs %}
+<ul id="page-tabs">
+	<li class="page"><a href="/childcount/rapports">Rapports</a></li>
+	<li class="page"><a href="/childcount/saisie">Saisie Enfant</a></li>
+	
+</ul>
+{% endblock %}
 
 {% block breadcrumbs %}
     <a href="/">{{ app.name }}</a>
diff --git a/apps/childcount/urls.py b/apps/childcount/urls.py
index 1d32d42..3777bbb 100755
--- a/apps/childcount/urls.py
+++ b/apps/childcount/urls.py
@@ -16,7 +16,9 @@ except AttributeError:
 urlpatterns = patterns('',
     admin_urls,
     url(r'^childcount/?$', views.index),
-    url(r'^childcount/reports?$', reports.reports),
+    url(r'^childcount/rapports?$', reports.reports),
+    url(r'^childcount/reports?$', reports.childcount),
+    url(r'^childcount/saisie?$', reports.saisie),
     (r'^report/(?P<report_name>[a-z\-\_]+)/(?P<object_id>\d*)$', reports.report_view),
     #last_30_days
     (r'^last_30_days/$', reports.last_30_days),
